daemon/containerd/image_push.go

```diff
@@ -100,16 +100,19 @@ func (i *ImageService) pushRef(ctx context.Context, targetRef reference.Named, m
 	}()
 
 	img, err := i.images.Get(ctx, targetRef.String())
-	if err != nil {
-		if cerrdefs.IsNotFound(err) {
-			return errdefs.NotFound(fmt.Errorf("tag does not exist: %s", reference.FamiliarString(targetRef)))
-		}
-		return errdefs.NotFound(err)
+	if cerrdefs.IsNotFound(err) {
```

Comment:
Why is the non-notfound err case no longer relevant?

Reply:
Looks like a mistake, thanks for spotting!